frontend/src/metabase/palette/shortcuts/question.ts

```diff
@@ -6,69 +6,69 @@ export const questionShortcuts = {
       return t`Toggle Notebook`;
     },
     shortcut: ["e"],
-    shortcutGroup: "question",
+    shortcutGroup: "question" as const,
```

Comment:
I don't think all this `as const` is necessary? not showing any type errors for me locally

Reply:
I get type errors because of what happens in the PaletteShortcutModal.tsx file.